### DISCUSSION TITLE
Select certificate if multiple linked?

### DISCUSSION THREAD

**Discussion by hannahshort** [indigo-iam/iam#528] [Q&A]
Hello IAM community, 

If a user has 2 linked certificates is it possible to select which cert is returned via the API? We see that it seems to be always the first one in the list. 

Thanks in advance! 
Hannah

**Comment by chaen** [comment]
After a short discussion with @giacomini, he suggested we run the VOMS import script without the `merging` option (I don't have access to the gitlab repo so can't point you at the line he showed me).
